### DISCUSSION TITLE
Does the calculation of the NVE using the deep potential conserve total energy?

### DISCUSSION THREAD

**Discussion by kudoryutaro** [deepmodeling/deepmd-kit#2684] [Q&A]
Does the calculation of the NVE using the deep potential conserve total energy?

**Comment by njzjz** [comment]
Yes, as the force is the negative gradient of the potential energy.

Note that if `sel` is not enough during simulations, the potential energy will be incorrect. See https://docs.deepmodeling.com/projects/deepmd/en/master/model/sel.html
